frontends/web/src/common/Annotation/CreateInterface.js

```diff
@@ -955,15 +880,6 @@ class CreateInterface extends React.Component {
                     </Row>
                   </Form>
                   <div className="p-2">
-                    {this.state.task.cur_round !==
```

Comment:
Maybe we can still keep some of this i.e if someone is talking to a random model because there is no model selected, we can keep a warning. In other words:
```
{!this.state.selectedModel && (
  <p style={{ color: "red" }}>
    WARNING: You are talking to a randomly selected model. If you would like to interact with a specific model for this task, please navigate to that model card and press `Interact`
  </p>
)}
```

Reply:
Yeah that's right!